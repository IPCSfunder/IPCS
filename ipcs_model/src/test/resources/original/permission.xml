<?xml version='1.0' encoding='UTF-8'?>
<dataset>

	<role role_objid = "1" name = "ADMIN"/>
	<role role_objid = "2" name = "CHILDREN"/>
	<role role_objid = "3" name = "STAFF"/>
	<role role_objid = "4" name = "PARENT"/>

	<permission PERMISSION_OBJID="1" NAME = "LOGIN"/>
	<permission PERMISSION_OBJID="2" NAME = "ADD_UPDATE_REMOVE_STUDENT"/>
	<permission PERMISSION_OBJID="3" NAME = "ADD_UPDATE_REMOVE_TEACHER"/>

	<role_permission role_permission_objid="1" ROLE_FK="1" PERMISSION_FK="1"/>
	<role_permission role_permission_objid="2" ROLE_FK="1" PERMISSION_FK="2"/>
	<role_permission role_permission_objid="3" ROLE_FK="1" PERMISSION_FK="3"/>
	<role_permission role_permission_objid="4" ROLE_FK="3" PERMISSION_FK="1"/>



</dataset>