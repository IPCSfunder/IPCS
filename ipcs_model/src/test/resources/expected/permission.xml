<?xml version='1.0' encoding='UTF-8'?>
<dataset>

	<role role_objid = "1" name = "ADMIN"/>
	<role role_objid = "2" name = "CHILDREN"/>
	<role role_objid = "3" name = "STAFF"/>
	<role role_objid = "4" name = "PARENT"/>

	<permission NAME = "LOGIN"/>
	<permission NAME = "ADD_UPDATE_REMOVE_STUDENT"/>
	<permission NAME = "ADD_UPDATE_REMOVE_TEACHER"/>
	<permission NAME = "ADD_UPDATE_REMOVE_ACTIVITY"/>

	<role_permission ROLE_FK="1" />
	<role_permission ROLE_FK="1" />
	<role_permission ROLE_FK="1" />
	<role_permission ROLE_FK="3" />
</dataset>